project-base/.env

```diff
@@ -14,5 +14,23 @@
 # https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
 
 IGNORE_DEFAULT_ADMIN_PASSWORD_CHECK='0'
-REDIS_PREFIX=''
+DATABASE_HOST=postgres
+DATABASE_PORT=5432
+DATABASE_NAME=shopsys
+DATABASE_USER=root
+DATABASE_PASSWORD=root
```

Comment:
Why are we do not connection string instead of those parameters? 

We would really appreciate this because our projects have a lot of ENV variables and we are lost with it. Moreover, Symfony is propagating connection strings and there is not any benefit for do not use connection string

Please consider using it ❤️

Reply:
We have considered it and decide to not implement it now. We do not want to change the behavior except passing parameters.

I personally like configuring connection by setting each parameter separate becouse I cannot remember the schema of connection string so I have to google it every time.

Reply:
Redundats were removed in 2f196da2d7881ea636a8c0972216fb5f366dea27

Reply:
I would like to have this feature in next major release. Is this possible? There can be comment about "How to use connection string". 

Can you create an issue for that? Or can we discuss it, because for us this little change has really big advantage

Reply:
Ok, I have created an issue https://github.com/shopsys/shopsys/issues/2072.

I am resolving this.